### PR TITLE
UIU-1535 Permission "Users: User loans anonymize"

### PR DESCRIPTION
**Pull request by axelhunn**
# Description
Purpose: create permission for users to manually anonymize loans.
- add new permission "Users: User loans anonymize"
- show `Anonymize all loans` button on a user's closed loans only if user got that permission
# Approach
- add a permission to `package.json`, name in English
- show the `Anonymize all loans` button conditionally, depending on the permission
# Issue
https://issues.folio.org/browse/UIU-1535

### CODE REVIEW

**Review by aliaksei-chumakou — APPROVED** [review]

**Review by uladislausamets — APPROVED** [review]

**Review by AndreiShumski2 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -611,6 +611,14 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "ui-users.loans.anonymize",
+        "displayName": "Users: User loans anonymize",
+        "subPermissions": [
+          "ui-users.view"
```
Comment (NikitaSedyx) [inline package.json:618]: I don't think you should inherit view permission, please read ticket scenarios and permissions `view` introduces

Comment (axelhunn) [reply package.json:618]: My mistake, forgot to get rid of subPermissions

File: package.json [hunk 2]
```diff
@@ -611,6 +611,11 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "ui-users.loans.anonymize",
+        "displayName": "Users: User loans anonymize",
+        "visible": true
```
Comment (NikitaSedyx) [inline package.json:617]: does it require any BE permissions?

**Comment by aliaksei-chumakou** [comment]
@axelhunn FYI, It's not the folio pattern - to include code refactoring in the same PR. It's better to merge code refactoring before, so PR contains only changes related to the ticket

**Review by aliaksei-chumakou — APPROVED** [review]

**Review by AndreiShumski2 — APPROVED** [review]

**Review by uladislausamets — APPROVED** [review]

**Comment by axelhunn** [comment]
> @axelhunn FYI, It's not the folio pattern - to include code refactoring in the same PR. It's better to merge code refactoring before, so PR contains only changes related to the ticket

Thanks, I thought fixing lint issues is not much of a refactoring to make separate PR

**Pull request merged**